python/cudf/cudf/tests/test_json.py

```diff
@@ -1006,3 +1006,95 @@ def test_json_round_trip_gzip():
         fo.seek(loc)
         written_df = cudf.read_json(fo, orient="records", lines=True)
     assert_eq(written_df, df)
+
+
+@pytest.mark.parametrize(
+    "jsonl_string",
+    [
+        # simple list with mixed types
+        """{"a":[123, {"0": 123}], "b":1.0}\n {"b":1.1}\n {"b":2.1}""",
```

Comment:
Why do these have newlines _and_ spaces separating the lines? Should the spaces be removed?
```suggestion
        """{"a":[123, {"0": 123}], "b":1.0}\n{"b":1.1}\n{"b":2.1}""",
```

Reply:
Spaces does not matter.